src/vs/workbench/contrib/debug/browser/rawDebugSession.ts

```diff
@@ -497,6 +497,22 @@ export class RawDebugSession implements IDisposable {
 		return Promise.reject(new Error('goto is not supported'));
 	}
 
+	async setInstructionBreakpoints(args: DebugProtocol.SetInstructionBreakpointsArguments): Promise<DebugProtocol.SetInstructionBreakpointsResponse | undefined> {
+		if (this.capabilities.supportsDisassembleRequest) {
```

Comment:
this must check the `supportsInstructionBreakpoints` capability (which is independent from `supportsDisassembleRequest`).
Make sure that a debug adapter with `supportsDisassembleRequest: true` and `supportsInstructionBreakpoints: false` still works.

Reply:
I've pushed a fix for that

Reply:
Thanks, I was just about to fix it.